xcell/tests/test_mapper_base_planck.py

```diff
@@ -0,0 +1,61 @@
+import xcell as xc
```

Comment:
Can you have a look at the new tests and see if you're still worried?

Reply:
I still think that it should be good to have some tests for the CIB and SMICA. There are none for those. In addition, I still think there should be a better way of testing the `mapper_planck_base`. Now, if, for instance, `get_nl_coupled` fails, you will not be able to say, directly, if it fails because of some change in `mapper_planck_base` or because `_get_hm_maps` is wrong in one of the child mappers.

Reply:
See new version. Note that `get_nl_coupled` (and many of the others) cannot be tested through the base class, since the base class throws `NotImplementedError`s for some of the intermediate methods.

Reply:
Add a test for `_get_hm_maps` and everything else is ok!

Reply:
Done!